arelle/ModelManager.py

```diff
@@ -73,7 +75,7 @@ def __init__(self, cntlr: Cntlr):
         self.abortOnMajorError = False
         self.collectProfileStats = False
         self.loadedModelXbrls = []
-        self.customTransforms = None
+        self.customTransforms: dict[QName, Callable[[str], str]] | None = None
```

Comment:
Seems like this should/could default to an empty dict instead?

Reply:
That could cause additional loads when there aren't any custom transforms (can't tell if they haven't been loaded yet or there just aren't any)? Anyway, I'm only doing the minimal touching of `ModelManager` while working on formula

Reply:
👍 with what you have, just getting at having collection set to null in general is an anti-pattern but not specifically calling for that to be addressed here. Perhaps `bool customTransformsIsSet` could be a work around? It seems odd longterm that it should be loading custom transforms in multiple places within the flow